bin/reth/src/node/mod.rs

```diff
@@ -202,3 +211,33 @@ fn init_genesis<DB: Database>(db: Arc<DB>, genesis: Genesis) -> Result<H256, ret
     tx.commit()?;
     Ok(hash)
 }
+
+// TODO: This should be based on some external config
+fn network_config<DB: Database>(
+    db: Arc<DB>,
+    chain_id: u64,
+    genesis_hash: H256,
+    disable_discovery: bool,
+) -> NetworkConfig<ProviderImpl<DB>> {
```

Comment:
this seems redundant because there's already an existing network_config function in config, so I don't understand why we need this.

Reply:
Sorry I messed it up when I rebased. I’ll fix it in a little bit.

Reply:
Should be all better now I just redid it because I messed up a bunch of stuff when I rebased. I'll do better next time. Thanks for your patience.